modules/imgproc/src/convhull.cpp

```diff
@@ -251,19 +251,18 @@ void convexHull( InputArray _points, OutputArray _hull, bool clockwise, bool ret
                 br_count = MIN( br_count, 2 );
             }
         }
-
-        for( i = 0; i < bl_count-1; i++ )
-            hullbuf[nout++] = int(pointer[bl_stack[i]] - data0);
-        for( i = br_count-1; i > 0; i-- )
-            hullbuf[nout++] = int(pointer[br_stack[i]] - data0);
+        for(int i = 0; i < bl_count-1; i++ )
+            hullbuf[nout++] = int(&pointer[bl_stack[i]] - pointer);
+        for(int i = br_count-1; i > 0; i-- )
+            hullbuf[nout++] = int(&pointer[br_stack[i]] - pointer);
```

Comment:
`&pointer[bl_stack[i]] - pointer` is just `bl_stack[i]`, right?

Reply:
nice catch! thx